### PR TITLE
THEMES-1079 Update the subheadline font to 20px level 6

### PR DESCRIPTION
**Pull request by Yoriichi0shiki**
## Description

- V2 Subheadline | Arc Block - Update the subheadline font to 20px level 6 instead of level 3

image

## Jira Ticket

- [THEMES-1079](https://arcpublishing.atlassian.net/browse/THEMES-1079)

## Acceptance Criteria

Subheadline font should be at level 6

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout {branch name}`
2. Run fusion repo with linked blocks `npx fusion start -f -l {blocks to link}`

### Before

Subheadline font should be at level 6

## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [ ] Confirmed all the test steps a reviewer will follow above are working.
- [ ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [ ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ ] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-1079]: https://arcpublishing.atlassian.net/browse/THEMES-1079?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by nschubach** [comment]
@malavikakoppula : was there a specific reason you marked this "do not merge" previously?

**Comment by malavikakoppula** [comment]
> @malavikakoppula : was there a specific reason you marked this "do not merge" previously?

For some reason, Styles are still not matching to the heading-level-6 designs.

File: .storybook/themes/news.scss [hunk 1]
```diff
@@ -2268,9 +2268,9 @@
 					grid-template-columns: 2fr 1fr,
 				),
 				subheadline: (
-					font-size: var(--heading-level-3-font-size),
-					font-weight: var(--heading-level-3-font-weight),
-					line-height: var(--heading-level-3-line-height),
+					font-size: var(--heading-level-5-font-size),
```
Comment (malavikakoppula) [inline .storybook/themes/news.scss]: change this to `font-size: var(--heading-level-6-font-size)`

File: .storybook/themes/news.scss [hunk 2]
```diff
@@ -2268,9 +2268,9 @@
...
+					font-weight: var(--heading-level-7-font-weight),
```
Comment (malavikakoppula) [inline .storybook/themes/news.scss]: `heading-level-7-font-weight` on `font-weight ` is not existed. Change to `heading-level-6-font-weight`

File: .storybook/themes/news.scss [hunk 3]
```diff
@@ -2268,9 +2268,9 @@
...
+					line-height: var(--heading-level-5-line-height),
```
Comment (malavikakoppula) [inline .storybook/themes/news.scss]: same as above, change to `heading-level-6-line-height`

File: blocks/subheadline-block/themes/news.json [hunk 4]
```diff
@@ -2,9 +2,9 @@
 	"subheadline": {
 		"styles": {
 			"default": {
-				"font-size": "var(--heading-level-3-font-size)",
-				"font-weight": "var(--heading-level-3-font-weight)",
-				"line-height": "var(--heading-level-3-line-height)"
+				"font-size": "var(--heading-level-5-font-size)",
+				"font-weight": "var(--heading-level-7-font-weight)",
+				"line-height": "var(--heading-level-5-line-height)"
```
Comment (malavikakoppula) [inline blocks/subheadline-block/themes/news.json]: "font-size": "var(--heading-level-6-font-size)",
"font-weight": "var(--heading-level-6-font-weight)",
"line-height": "var(--heading-level-6-line-height)"

**Review by malavikakoppula — CHANGES_REQUESTED** [review]
For font-size, font-weight and line-height have to refer to heading-level-6

**Review by malavikakoppula — APPROVED** [review]
LGTM

**Comment by pardistaherzadeh** [comment]
@alvajardila It is approved and ready to be merged.

**Pull request merged**